yoda_dbt2looker/parser.py

```diff
@@ -250,25 +271,25 @@ def _extract_measures_models(
             exposure_model_views.update(_extract_all_refs(measure.sql))
 
 
-def _extract_calculated_dimensions_models(
+def _extract_models(
```

Comment:
Not sure that the name explains what the method does - the models you are referring are actually more custom members of a model - Consider providing a name that will explain better the method